install/kubernetes/cilium/values.yaml

```diff
@@ -1903,9 +1903,9 @@ envoy:
   image:
     override: ~
     repository: "quay.io/cilium/cilium-envoy"
-    tag: "v1.26.4-bad30dbe3f4c33c4a6e6747236f7c6d84b380d0d"
+    tag: "v1.26.4-5a76016dde9a7b4d537f37e3ef84593ea1af87c7"
     pullPolicy: "Always"
-    digest: "sha256:ec4f5a3b9504288b4de1db00893bd0ef79187c501a63e08bce11e2fb3de88920"
+    digest: "sha256:dbcb20bb208eb0031991f5c234eb6de567f95ebd814520dcbfda868b7c1af210"
```

Comment:
was surprised to see changes not reflected in `install/kubernetes/cilium/values.yaml.tmpl` but theses values are templatized so that's why CI is not complaining.

Reply:
I stumbled on this in a prevous PR to. The answer is that those digests are the sole reason we have the whole `install/kubernetes/cilium/values.yaml.tmpl` plumbing in the first place. CI does not require that the template is identical to the `values.yaml`, it only requires that the output generated from `values.yaml.tmpl` is equivalent to `values.yaml`.